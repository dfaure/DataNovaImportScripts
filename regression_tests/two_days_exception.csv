17746A;LE ROBERT VERT PRE BP;2021-01-28;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-29;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-08;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-04;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-14;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2020-12-29;07:30-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-19;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-20;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-13;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-27;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-03;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-12;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-15;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-18;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-11;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-04;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-05;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-10;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-07;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-26;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2020-12-30;07:30-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-16;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2020-12-28;07:30-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-21;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-23;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-30;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-06;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-25;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-22;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-09;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-18;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-01;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-22;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2020-12-31;07:30-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-01-02;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-10;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-09;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-19;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-25;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-16;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-27;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-20;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-11;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-08;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-03;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-01;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-06;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-26;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-15;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-13;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-22;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-18;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-11;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-04;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-15;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-05;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-20;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-02;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-30;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-28;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-23;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-24;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-05;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-09;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-17;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-13;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-12;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-21;FERME;;;;;;;;;;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-12;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-25;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-08;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-26;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-02;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-23;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-17;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-02-06;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-16;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-24;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-27;07:30-11:30;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-19;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-31;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
17746A;LE ROBERT VERT PRE BP;2021-03-29;07:30-12:15;;;;;;;11:00;11:00;11:00;;FERME;;;;;;;
