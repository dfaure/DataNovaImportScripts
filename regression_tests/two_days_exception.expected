17746A|LE ROBERT VERT PRE BP|Mo-Fr 07:30-12:15; Sa 07:30-11:30; PH off; 2020 Dec 28-31 07:30-12:00; 2021 Jan 02,09,Feb 15-17 off
